workflows/workflow/cp_process.smk

```diff
@@ -1,26 +1,29 @@
 import glob
-
+from cytosnake.helpers import helper_funcs as hf
 
 # obtaining plate_ids
 PLATE_IDS = glob_wildcards("data/{id}.sqlite").id
 
 
 # importing DAGs
-include: "rules/preprocessing.smk"
-include: "rules/feature_select.smk"
+include: "../rules/common.smk"
+include: "../rules/preprocessing.smk"
+include: "../rules/feature_select.smk"
 
 
 # appending logs
 # LOG_NAMES = glob_wildcards("logs/{log_name}.log").log_name
```

Comment:
Should this commented out code still be here?

Reply:
oops forgot to tidy up! Thanks!